index.html

```diff
@@ -0,0 +1,155 @@
+        </div>
+
+        <div class="flex-row justify-center align-center">
+          <div class="mr-2"><a href="/authentication/signin/signin.html">Login</a></div>
+          <div class="component-image-rounded user-sm-image">
+            <img
+              src="/images/avatar.svg"
+              alt="user-img"
+            />
+          </div>
+        </div>
+      </div>
+      <div class="main-container">
+        <div class="sidebar ml-2 p-2">
+          <ul class="sidebar-unorderd-list">
+            <li class="list-style-none">
+              <button class="active sidebar-list-item font-semibold text-xs p-2 mb-2">
+                Dashboard
+              </button>
+            </li>
+            <li class="list-style-none">
+              <button class="sidebar-list-item font-semibold text-xs p-2 mb-2">
+               Support
+              </button>
+            </li>
+            <li class="list-style-none">
+              <button class="sidebar-list-item font-semibold text-xs p-2">
+              Notification
+              </button>
+            </li>
+          </ul>
+          <div class="logout font-semibold text-xs">
+            <a href="">Logout</a>
+          </div>
+        </div>
+        <div class="main-content-container flex-col p-2">
+          <div class="main-content-section1-container flex-row mb-1">
+            <div class="user-image-container component-image-rounded">
+              <img
+                class="user-image"
+                src="/images/avatar.svg"
+                alt="user-image"
+              />
+            </div>
+            <div class="user-details-container">
+              <div class="user-details-section1-container">
+                <h2>Priya Patel</h2>
+                <small>priya@gmail.com</small>
+              </div>
+              <div class="user-details-section2-container mt-2 flex-row">
+                <div class="total-score-container flex-row">
+                  <i class="fa-solid fa-flag text-md"></i>
+                  <div class="flex-col align-center">
+                    <h2>27</h2>
+                    <p>Total score</p>
+                  </div>
+                </div>
+                <div class="total-score-container flex-row">
+                  <i class="fa-solid fa-clock text-md"></i>
+                  <div class="">
+                    <h2>27min</h2>
+                    <p>Total time</p>
+                  </div>
+                </div>
+                <div class="total-score-container flex-row">
+                  <i class="fa-solid fa-circle-check text-md"></i>
+                  <div class="flex-col align-center">
+                    <h2 class="">200</h2>
+                    <p>Answers</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="main-content-section2-container p-1 flex-row">
+            <div class="main-content-section2-card-container">
```

Comment:
Same as above too long class names.

Reply:
But if this className is making sense then we can give bigger name to it as I have already given main-content-container to one of the div's hence can't use that so used this . 
And we have to take care of better naming convention too hence used as it is making sense by the name itself.

Reply:
Use classes from component library whenever possible